apps/marketplace/pages/BuyerDashboardPage.js

```diff
@@ -1,26 +1,77 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { withRouter, Switch, Route, BrowserRouter } from 'react-router-dom'
 import BuyerDashboardHeader from 'marketplace/components/BuyerDashboard/BuyerDashboardHeader'
-import BuyerDashboardMyBriefs from 'marketplace/components/BuyerDashboard/BuyerDashboardMyBriefs'
-import BuyerDashboardTeamBriefs from 'marketplace/components/BuyerDashboard/BuyerDashboardTeamBriefs'
-import BuyerDashboardTeamOverview from 'marketplace/components/BuyerDashboard/BuyerDashboardTeamOverview'
+import BuyerDashboardAllBriefs from 'marketplace/components/BuyerDashboard/BuyerDashboardAllBriefs'
+import BuyerDashboardDraftBriefs from 'marketplace/components/BuyerDashboard/BuyerDashboardDraftBriefs'
+import BuyerDashboardLiveBriefs from 'marketplace/components/BuyerDashboard/BuyerDashboardLiveBriefs'
+import BuyerDashboardClosedBriefs from 'marketplace/components/BuyerDashboard/BuyerDashboardClosedBriefs'
 import { rootPath } from 'marketplace/routes'
 
-const BuyerDashboardPage = props => (
-  <BrowserRouter basename={`${rootPath}/buyer-dashboard`}>
-    <div>
-      <BuyerDashboardHeader {...props} />
-      <article role="main">
-        <Switch>
-          <Route exact path="/" render={() => <BuyerDashboardMyBriefs {...props} />} />
-          <Route path="/team-briefs" render={() => <BuyerDashboardTeamBriefs {...props} />} />
-          <Route path="/team-overview" render={() => <BuyerDashboardTeamOverview {...props} />} />
-        </Switch>
-      </article>
-    </div>
-  </BrowserRouter>
-)
+export class BuyerDashboardPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      briefCounts: {
+        closed: 0,
+        draft: 0,
+        live: 0,
+        withdrawn: 0
+      }
+    }
+  }
+
+  dashboardLoaded(data) {
+    this.setState({
+      briefCounts: data.briefCount,
+      organisation: data.organisation
+    })
+  }
+
+  render() {
+    const props = this.props
+    return (
+      <BrowserRouter basename={`${rootPath}/buyer-dashboard`}>
+        <div>
+          <BuyerDashboardHeader
```

Comment:
Can you use the `PageHeader` and `PageNavigation` components here?

Reply:
updated